app/src/main/kotlin/no/nav/k9punsj/korrigeringinntektsmelding/KorrigeringInntektsmeldingRoutes.kt

```diff
@@ -132,7 +133,15 @@ internal class KorrigeringInntektsmeldingRoutes(
                     )?.let { return@RequestContext it }
                 }
 
-                val (validertSøknad, søknadFeil) = korrigeringInntektsmeldingService.validerSøknad(søknad)
+                val (validertSøknad, søknadFeil) = try {
+                    korrigeringInntektsmeldingService.validerSøknad(søknad)
+                } catch(e: Exception) {
```

Comment:
Finnes det ikke en global exception handler for å håndtere det der?

Reply:
Jo, det er jo då vi får "ukjent feil" .. Virker rart att vi får ukjent feil når det er forventet :D 
Her førsøker jag bara standardisere for o bli kvitt mappingfeilet i errorloggen.... var såher det såg ut i PSB

Reply:
Aah OK 👍